static/src/stylesheets/module/commercial/_paidfor.scss

```diff
@@ -1,6 +1,12 @@
 /* ==========================================================================
    Paid for component
    ========================================================================== */
+
+.fc-container .fc-container__inner--paidfor {
```

Comment:
Isn't `.fc-container__inner--paidfor` enough on its own? unless `.fc-container__inner` has already been nested elsewhere of course.